src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/template.json

```diff
@@ -28,7 +28,7 @@
     {
       "id": "openInEditor",
       "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
-      "description": "Opens the created view in the editor",
+      "description": "Opens Index.cshtml in the editor",
```

Comment:
This is also similar to `HomeController` and `ValueController`, the file that's generated will be determined by the user, `Index.cshtml` is not likely to make sense to the enduser.